packages/terra-framework-docs/src/terra-dev-site/doc/popup/example/PopupStandard.jsx

```diff
@@ -23,8 +23,20 @@ function PopupStandard() {
     setOpen(false);
   };
 
+  useEffect(() => {
+    if (open) {
+      document.querySelector('#root').setAttribute('inert', 'true');
+    } else {
+      document.querySelector('#root').removeAttribute('inert');
+    }
+  }, [open]);
```

Comment:
Can we simplify the above code ?
 
```suggestion
  useEffect(() => {
    document.querySelector('#root').setAttribute('inert', open);

    return () => {
      document.querySelector('#root').removeAttribute('inert');
    };
  }, [open]);
```